### DISCUSSION TITLE
MSVC编译器， Qt调试的时候，怎么显示Qt类型的实际变量值

### DISCUSSION THREAD

**Discussion by scially** [xmake-io/xmake#2914] [Q&A]
MSVC编译器， Qt调试的时候，怎么显示Qt类型的实际变量值

**Comment by waruqi** [comment]
这跟 xmake 有啥关系？

**Reply by scially** [reply]
就是xmake 能添加natvis文件么？在VSCode里，直接用xmake写Qt工程，调试时，QString这些变量是直接看不到值的

**Reply by waruqi** [reply]
还不支持，目前只有 vsxmake generator 支持带 natvis 文件的生成。
